README.md

```diff
@@ -245,13 +245,14 @@ Then similarly with `matrix-react-sdk`:
 
 Finally, build and start Riot itself:
 
-1. `git clone git@github.com:vector-im/riot-web.git`
+1. `git clone https://github.com/vector-im/riot-web.git`
 1. `cd riot-web`
 1. `git checkout develop`
 1. `npm install`
 1. `rm -r node_modules/matrix-js-sdk; ln -s ../../matrix-js-sdk node_modules/`
 1. `rm -r node_modules/matrix-react-sdk; ln -s ../../matrix-react-sdk node_modules/`
-1. `npm start`
+1. `cp -vi config.sample.json config.json` # and configure it
```

Comment:
for developing on riot-web this isn't really needed? I know I've never had to do it

Reply:
i didn't add this line, I just added the `cp` line

Reply:
`npm start` works fine on my server for running Riot with the built-in webserver

Reply:
I'm referirng to the CP line

Reply:
If you want to run a Riot instance with your own settings, you have to adapt the config here. maybe we should change the comment to 

    # and configure it, if needed

Reply:
I think we'd best leave it out. It'd be nicer to have an entire piece of documentation to explain the available options that can be configured.